src/analysis/raster/qgsrastercalculator.h

```diff
@@ -78,7 +81,8 @@ class ANALYSIS_EXPORT QgsRasterCalculator
     QgsRasterCalculator( const QString& formulaString, const QString& outputFile, const QString& outputFormat,
                          const QgsRectangle& outputExtent, const QgsCoordinateReferenceSystem& outputCrs, int nOutputColumns, int nOutputRows, const QVector<QgsRasterCalculatorEntry>& rasterEntries );
 
-    /** Starts the calculation and writes new raster
+    /**
+     * Starts the calculation and writes new raster
```

Comment:
more occasions where a leading `*` is added only to the first line of a multiline comment.